mlrun/utils/helpers.py

```diff
@@ -44,6 +44,35 @@
 DB_SCHEMA = "store"
 
 
+class StorePrefix:
```

Comment:
I suggest to move everything here `StorePrefix`, `is_store_uri`,`parse_store_uri`, `get_store_uri` to the `resources_store.py` file

Reply:
moved, kept `StorePrefix` there to avoid circular dep